### PR TITLE
Add folders to gitignore

### PR DESCRIPTION
**Pull request by renzodgc**

### CODE REVIEW

**Comment by zylo117** [comment]
@renzodgc Hi, can it work?

**Comment by renzodgc** [comment]
@zylo117 Hello! First of all sorry for the PR! My intention was to open it directed to a forked repo but I miss clicked :sweat_smile: 

Regarding your question, the training script is not very robust right now. We only succeeded in training a D3 model with apex but had issues with exploding gradients when training D2 and D1.

We'd be delighted to open a proper pull request to this repository in case we manage to make it work for the other configurations :slightly_smiling_face:
